plugins/TopPairMuonPlusJetsSelectionFilter.cc

```diff
@@ -301,9 +301,13 @@ bool TopPairMuonPlusJetsSelectionFilter::passesLooseMuonVeto() const {
 			}
 		}
 		return isZEvent == true;
-	} else
-		// Require only one loose muon, which is the signal muons
-		return vetoMuons_.size() < 2;
+	} 	
+	else {
+		if (nonIsolatedMuonSelection1_ || nonIsolatedMuonSelection2_){
+			return vetoElectrons_.size() < 1;
+		}
+		return vetoElectrons_.size() < 2;
```

Comment:
Should this not be `vetoElectrons_.size() < 1` (or `==0`) in all cases? We do not want any electrons in the muon selection.